uaclient/entitlements/repo.py

```diff
@@ -235,10 +213,42 @@ def process_contract_deltas(
                 apt.remove_auth_apt_repo(repo_filename, old_url)
         self.remove_apt_config()
         self.setup_apt_config()
+        if delta_packages:
```

Comment:
I don't know if we want to support that, but could there be a situation were we would require to uninstall the current packages and install the new ones from the delta ?

Reply:
@lucasmoura I was considering the same condition. I think if we wanted an explicity removal of packages across deltas, we would probably have to design that with the contract backend if required. Since most of our logic is only to supplement via additionalPackages directive, I'd probably want to see a "removePackages": directive or something similar to mandate that packages get removed. Right now we don't really have a use case for it, so I didn't want to put together a solution for something that "may" not be needed.